packages/suite/src/views/wallet/tokens/inactive-tokens/InactiveTokensTable.tsx

```diff
@@ -0,0 +1,226 @@
+                })
+                .catch(error => {
+                    console.error('Failed to load inactive tokens:', error);
+                    setAllInactiveTokens([]);
+                    setIsLoading(false);
+                    setHasError(true);
+                    dispatch(
+                        notificationsActions.addToast({
+                            type: 'error',
+                            error: 'Failed to load inactive tokens. Please try again later.',
+                        }),
+                    );
+                });
+        } else {
+            setAllInactiveTokens([]);
+            setIsLoading(false);
+        }
+    }, [account, dispatch]);
+
+    const filteredTokens = useMemo(() => {
+        if (!searchQuery) {
+            return allInactiveTokens;
+        }
+
+        const lowerCaseQuery = searchQuery.toLowerCase();
+
+        return allInactiveTokens.filter(
+            token =>
+                token.name?.toLowerCase().includes(lowerCaseQuery) ||
+                token.contract.toLowerCase().includes(lowerCaseQuery),
+        );
+    }, [searchQuery, allInactiveTokens]);
+
+    const handleActivateToken = (token: TokenInfo) => {
+        setTokenToActivate(token);
+    };
+
+    const closeActivateModal = () => setTokenToActivate(null);
+
+    // Only show for Stellar network
+    if (account.symbol !== 'xlm') {
+        return <NoTokens title={<Translation id="TR_INACTIVE_TOKENS_EMPTY" />} />;
+    }
+
+    if (isLoading) {
+        return <Loading />;
+    }
+
+    if (hasError) {
+        return <NoTokens title={<Translation id="TR_ERROR_LOADING_TOKENS" />} />;
+    }
+
+    if (filteredTokens.length === 0 && searchQuery) {
+        return <NoSearchResultsWrapped />;
+    }
+
+    if (filteredTokens.length === 0) {
+        return <NoTokens title={<Translation id="TR_INACTIVE_TOKENS_EMPTY" />} />;
+    }
+
+    return (
+        <Card paddingType="none" overflow="hidden">
+            <Table
+                margin={{ top: spacings.xs }}
+                colWidths={[
+                    { minWidth: '200px', maxWidth: '250px' },
+                    { minWidth: '140px', maxWidth: '250px' },
+                ]}
+                isRowHighlightedOnHover
+            >
+                <Table.Header>
+                    <Table.Row>
+                        <Table.Cell>
+                            <Translation id="TR_TOKEN" />
+                        </Table.Cell>
+                        <Table.Cell>
+                            <Translation id="TR_ISSUER" />
+                        </Table.Cell>
+                        <Table.Cell />
+                    </Table.Row>
+                </Table.Header>
+                <Table.Body>
+                    {filteredTokens.map(token => (
+                        <Table.Row key={token.contract}>
+                            <Table.Cell>
+                                <Row gap={spacings.xs}>
+                                    <AssetLogo
```

Comment:
Regarding icons from coingecko. Should we cut the `-1` when uploading images to data.trezor.io?

https://www.coingecko.com/en/coins/blend address `BLND-GDJEHTBE6ZHUXSWFI642DCGLUOECLHPF3KSXHPXTSTJ7E3JF6MQ5EZYY-1`

https://data.trezor.io/suite/icons/coins/stellar--BLND-GDJEHTBE6ZHUXSWFI642DCGLUOECLHPF3KSXHPXTSTJ7E3JF6MQ5EZYY-1@2x.webp

<img width="613" height="87" alt="Screenshot 2025-10-30 at 7 52 49" src="https://github.com/user-attachments/assets/05f9abd8-cdd3-4150-ad3e-71df4dbaf9b2" />